doc/API.md

```diff
@@ -31,12 +31,25 @@ Display general information about the current daemon state.
 | `sync`        | float   | The synchronization progress as percentage (`0 < sync < 1`)     |
 | `version`     | string  | Version following the [SimVer](http://www.simver.org/) format   |
 
+
+### `getdepositaddress`
+
+Get an address to build a deposit transaction.
+
+#### Response
+
+| Field         | Type   | Description                                                 |
+| ------------- | ------ | ----------------------------------------------------------- |
+| `address`     | string | An address for the N-of-N multisig deposit script           |
```

Comment:
Provide the address format

Reply:
We only ever support Segwit v0, don't you think that's redundant ?

Reply:
Will add a commit specifying this for the whole doc

Reply:
Done